grayskull/license/discovery.py

```diff
@@ -61,7 +61,7 @@ def match_license(name: str) -> dict:
     all_licenses = get_all_licenses_from_spdx()
     name = re.sub(r"\s+license\s*", "", name.strip(), flags=re.IGNORECASE)
 
-    best_matches = process.extractBests(name, _get_all_license_choice(all_licenses))
```

Comment:
extract and extractBests in FuzzyWuzzy are the same with the difference that extractBests allows the use of score_cutoff which is not possible with extract.
RapidFuzz is based on an older version of FuzzyWuzzy that did not have extractBests, but has the score_cutoff argument for extract. So extract does the same extractBests does